lib/msal-node/src/network/HttpClient.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { INetworkModule, NetworkRequestOptions } from '@azure/msal-common';
+import { HttpMethod } from './../utils/Constants';
+import axios from 'axios';
+
+/**
+ * This class implements the Fetch API for GET and POST requests. See more here: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
+ */
+export class HttpClient implements INetworkModule {
+    /**
+     * Axios library for REST endpoints - Get request
+     * @param url
+     * @param headers
+     * @param body
+     */
+    async sendGetRequestAsync<T>(url: string, options?: NetworkRequestOptions): Promise<T> {
+        // setHeaders
+        this.setHeaders(options);
+
+        // axios config
+        const config = {
+            method: HttpMethod.GET,
+            url: url
+        };
+
+        // GET call
+        const response = await axios(config);
+        return await response as unknown as T;
```

Comment:
this line confuses me since we are awaiting response already.

should it be return response as T? 
and what is unknown for?

Reply:
okay here is the dilemma; msal-browser is doing `return await response.json() as T` which fails for msal-node compilation. I tried `return await response as T` : same issue. I want to investigate further to see what is ideal here, but currently this is the only way I could get it to work.

Reply:
Should be able to do return `response.data as T`.

Reply:
That is the next thought I had :)

Reply:
I want to execute this once for error responses and then update!

Reply:
Done. Please review.